pkg/controller/complianceremediation/complianceremediation_controller.go

```diff
@@ -236,23 +239,72 @@ func (r *ReconcileComplianceRemediation) reconcileRemediation(instance *compv1al
 				"Make sure the CRD is installed: %w", err)
 	} else if kerrors.IsNotFound(err) {
 		if instance.Spec.Apply {
+			// Going through remediation list, to make sure all the related
+			// remediations objects are set to apply
+			err := r.setRemediations(instance, objectLogger, true)
+			if err != nil {
+				return fmt.Errorf("failed to set related remediations to apply: %w", err)
+			}
+			err = r.createRemediation(obj, objectLogger)
+			if err != nil {
+				return fmt.Errorf("failed to create remediation: %w", err)
+			}
 			instance.AddOwnershipLabels(obj)
-			return r.createRemediation(obj, objectLogger)
+			return nil
```

Comment:
Hmm, isn't `instance.AddOwnershipLabels` now a no-op? It used to be called before `createRemediation` but now it's moved after, so we'd probably lose the labels with this patch?